website/i18n/en.json

```diff
@@ -89,6 +97,10 @@
         "title": "Platform modules",
         "sidebar_label": "Platform modules"
       },
+      "pricing-models": {
+        "title": "SaaS pricing models explained",
+        "sidebar_label": "Pricing Models"
+      },
```

Comment:
I still think there's no need for the hyphen in "Extra resources"

plus be consistent with the capital letters